app/javascript/layouts/Layout.tsx

```diff
@@ -170,8 +170,12 @@ const Layout = (props: LayoutProps) => {
           imageOnly={true}
           mobileText={true}
           logoWidth={"medium"}
+          logoClass="translate"
           menuLinks={getMenuLinks(!!profile, signOut)}
-          strings={{ skipToMainContent: t("t.skipToMainContent") }}
+          strings={{
+            skipToMainContent: t("t.skipToMainContent"),
+            logoAriaLable: t("t.dahliaSanFranciscoHousingPortal"),
```

Comment:
this one we may need to add to the [glossary](https://sfgovdt.jira.com/wiki/spaces/HOUS/pages/3189538929/Updating+DAHLIA+translations+using+Phrase+and+Freeway#SF.gov-Glossary)